Formula/apache-spark.rb

```diff
@@ -18,6 +18,6 @@ def install
   end
 
   test do
-    system "#{bin}/spark-shell <<<'sc.parallelize(1 to 1000).count()'"
+    system "echo 'sc.parallelize(1 to 1000).count()' | #{bin}/spark-shell"
```

Comment:
Oh, dang.  You might have to use `IO.popen`; [`ircii.rb`](https://github.com/Homebrew/homebrew-core/blob/master/Formula/ircii.rb) is an example.